uint8 triangle
uint8 circle
uint8 cross
uint8 square

uint8 d_up
uint8 d_right
uint8 d_down
uint8 d_left

uint8 right_trigger
uint8 right_bumper
uint8 left_trigger
uint8 left_bumper

uint8 start
uint8 select

float64 left_joy_x
float64 left_joy_y
uint8 left_joy_in


float64 right_joy_x
float64 right_joy_y
uint8 right_joy_in

